FusionIIIT/applications/alumni/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+# Create your models here.
+
+class Job(models.Model):
```

Comment:
Add a link field so that if an alumni wants to share just the link, they can simply add a link to the job posting without having to spend time on adding job descriptions.

Reply:
Create an ID field which would be a Primary Key. There is no eligible field for Primary Key right now.